app/ts/components/pages/Home.tsx

```diff
@@ -123,13 +108,8 @@ function FirstCard(param: FirstCardParams) {
 			<div class = 'card-content'>
 				{ param.useSignersAddressAsActiveAddress || !param.simulationMode ?
 					<p style = 'color: var(--text-color); text-align: left; padding-bottom: 10px'>
-						{ param.signerName === 'NoSigner'
-							? 'Retrieving address from a signer'
-							: <>Retrieving from&nbsp;<SignersLogoName signerName = { param.signerName } /></>
-						}
-						{ param.signerAccounts !== undefined && param.signerAccounts.length > 0 && param.tabIconDetails.icon !== ICON_NOT_ACTIVE ? <span style = 'float: right; color: var(--primary-color);'> CONNECTED </span> :
-							param.tabIconDetails.icon === ICON_SIGNING || param.tabIconDetails.icon === ICON_SIGNING_NOT_SUPPORTED ? <span style = 'float: right; color: var(--negative-color);'> NOT CONNECTED </span> : <></>
-						}
+						{ param.tabState === undefined || param.tabState?.signerName === 'NoSigner' ? <></> : <>Retrieving from&nbsp;<SignersLogoName signerName = { param.tabState.signerName } /></> }
+						{ param.tabState?.signerConnected ? <span style = 'float: right; color: var(--primary-color);'> CONNECTED </span> : <span style = 'float: right; color: var(--negative-color);'> NOT CONNECTED </span> }
```

Comment:
spaces within spans (inline element) will be negated anyways
```suggestion
						{ param.tabState?.signerConnected ? <span style = 'float: right; color: var(--primary-color);'>CONNECTED</span> : <span style = 'float: right; color: var(--negative-color);'>NOT CONNECTED</span> }
```